probe/endpoint/conntrack.go

```diff
@@ -250,6 +252,53 @@ func removeInplace(s, sep []byte) []byte {
 	return s[:len(s)-len(sep)]
 }
 
+func decodeTwoTuples(line []byte, f *flow) error {
```

Comment:
The function name isn't self-explanatory (to me at least). Would you mind renaming it to a more descriptive name and adding a comment? e.g. `decodeFlowKeyValues`, `addFlowKeyValues`, `decodeFlowKeySuffix`....